discordbot/fetch_dives.py

```diff
@@ -0,0 +1,72 @@
+from ansitable import ANSITable
+
+
+def _clean_line(line):
+	return [
+		# Return every token
+		token.strip()
+		# delimited by `|`
+		for token in line.split("|")
+		# if it is not empty
+		if token
+	]
+
+def _format_warning(w):
+	# No warning, no special highlighting
+	if w == "None":
+		return "{0}".format(w)
+	# No warning known yet, highlight in white
+	elif w == "TBA":
+		return "({0})".format(w)
+	# Warning found, highlight in red
+	else:
+		return "[{0}]".format(w)

```

Comment:
`.format` is an oldschool way of using template string and should be replaced with f-strings:
```py
return f"{w}"
...
return f"({w})"
...
return f"[{w}]"
```

Reply:
If you look closely at the first return, all it does is return `None` if it is `None`. This line can be replaced with
```py
return "None"
```
Similar logic is for second line:

```py
return "(TBA)"
```

I thought that `w` means word, while in fact it means `warning`. It should be renamed to that.

There are many way to rewrite this function for scalability (maps, switch case (py3.10)), but if you would want to still use if/else, this is how this function could look like based on my previous comments:

```py
# Highlights warning cell
def __format_warning(warning):
	if warning == "None":
           return "None"
	elif warning == "TBA":
           return "(TBA)" # highlight in white
	else:
	   return f"[{warning}]" # highlight in red
```